cli.ts

```diff
@@ -76,6 +77,22 @@ async function cli(args: ParsedArgs) {
     process.exit(0)
   }
 
+  if (
+    args._[0] === "config" &&
+    args._[1] === "set" &&
+    args._[2] === "fake-server"
+  ) {
+    const randomstring = randomBytes(5).toString("hex")
```

Comment:
8**5 is the number of combinations, it's a bit low and could mean a "collision" occurs, let's increase it to 8**8

```suggestion
    const randomstring = randomBytes(8).toString("hex")
```